data/ecosystems/z/zkswap-finance.toml

```diff
@@ -0,0 +1,7 @@
+# Ecosystem Level Information
+title = "zkSwap Finance"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/ZkSwapFinance"]
```

Comment:
this is not a github org url, it's a user url

Reply:
Thank you, we have revised to 3 repositories.